script.js

```diff
@@ -1 +1,19 @@
+const todoInput = document.getElementById("todo-input");
+const addTodoBtn = document.getElementById("add-todo-btn");
+const taskListContainer = document.getElementById("task-list-container");
+const taskList = document.getElementById("task-list");
 
+addTodoBtn.addEventListener("click", addTodo);
+
+/**
+ * Adding todo, clearing input after adding
+ */
+function addTodo() {
+  const newTodo = document.createElement("li");
+  newTodo.innerHTML = todoInput.value;
+  const editIcon = document.createElement("span");
+  newTodo.style.listStyleType = "circle";
+  taskList.append(newTodo);
+
+  todoInput.value = "";
+}
```

Comment:
1. Rename as suggested
2. This function is called when the add button is clicked. So it should deligate the works that needed to be done to create a todo to other functions
```suggestion
let todos = [] // this list will hold todo objects

function renderTodos(){
  const todoTitle = todoInput.value
  const $todo = createTodo(todoTitle)
 // render todos list here by iterating over the todos array
}

function onAddTodo() {
  
  
  renderTodos()
  
  todoInput.value = "";
}

function createTodo(todoTitle){
  const $todo = document.createElement("li");
  
  $todo.innerHTML = todoInput.value;
  $todo.style.listStyleType = "circle";
  
  return $todo
}
```

Reply:
Thank you for the suggestion bhaia, I have refactored the code to follow your suggested structure. Kindly review again.